07_cmd_lab/cmdlab.txt

```diff
@@ -3,13 +3,43 @@ Questions online: https://www.tocode.co.il/bundles/unix1/lessons/cmdlab
 
 1. Login to your UNIX account. How do you find:
   - your current shell?
+echo $0
   - your default shell?
+echo $SHELL
   - a list of all shells currently installed on the machine?
+cat /etc/shells
+
 
 עברית שפה קשה. בעקרון המעטפת הנוכחית שלי היא bash
 
 2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
 
+Password file,
+Seven fiels:
+
+	·   login name
+
+       ·   optional encrypted password
+
+       ·   numerical user ID
+
+       ·   numerical group ID
+
+       ·   user name or comment field
+
+       ·   user home directory
+
+       ·   optional user command interpreter
+
 3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
 
+ls -s
```

Comment:
The `-s` option shows size in blocks. It's usually more effective to show/sort by size in bytes. Try `ls -lS /etc` for a nicer result